clientv3/txn_test.go

```diff
@@ -98,4 +100,6 @@ func TestTxnPanics(t *testing.T) {
 			t.Errorf("#%d: did not panic, wanted panic %s", i, tt.err)
 		}
 	}
+
+	testutil.WaitPanic()
```

Comment:
i do not quite understand why do we need this. Shouldnt we already catch all panics with the defer function? We actually wait for the defer already, no?